### PR TITLE
OCPBUGS-4009: Console operator should report ConsolePlugins as relate…

### PR DESCRIPTION
**Pull request by TheRealJon**
…d objects

### CODE REVIEW

File: pkg/console/starter/starter.go [hunk 1]
```diff
@@ -393,6 +393,24 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		controllerContext.EventRecorder,
 	)
 
+	// Show all CosolePlugin instances as related objects
```
Comment (jhadvig) [inline pkg/console/starter/starter.go:396]: nit
```suggestion
	// Show all ConsolePlugin instances as related objects
```

**Review by jhadvig — APPROVED** [review]
Thanks @TheRealJon 
One small nit, otherwise LGTM 👍

**Review by jhadvig — APPROVED** [review]
/lgtm

**Comment by jhadvig** [comment]
```
	clusteroperator/kube-apiserver is degraded because NodeControllerDegraded: The master nodes not ready: node "ip-10-0-183-30.ec2.internal" not ready since 2022-12-20 23:13:59 +0000 UTC because NodeStatusUnknown (Kubelet stopped posting node status.)
	clusteroperator/kube-controller-manager is degraded because NodeControllerDegraded: The master nodes not ready: node "ip-10-0-183-30.ec2.internal" not ready since 2022-12-20 23:13:59 +0000 UTC because NodeStatusUnknown (Kubelet stopped posting node status.)
	clusteroperator/kube-scheduler is degraded because NodeControllerDegraded: The master nodes not ready: node "ip-10-0-183-30.ec2.internal" not ready since 2022-12-20 23:13:59 +0000 UTC because NodeStatusUnknown (Kubelet stopped posting node status.)
	clusteroperator/machine-config is not available (Cluster not available for [{operator 4.13.0-0.ci.test-2022-12-20-222825-ci-op-38rl2m8k-latest}]: failed to apply machine config daemon manifests: error during waitForDaemonsetRollout: [timed out waiting for the condition, daemonset machine-config-daemon is not ready. status: (desired: 6, updated: 6, ready: 4, unavailable: 2)]) because Failed to resync 4.13.0-0.ci.test-2022-12-20-222825-ci-op-38rl2m8k-latest because: failed to apply machine config daemon manifests: error during waitForDaemonsetRollout: [timed out waiting for the condition, daemonset machine-config-daemon is not ready. status: (desired: 6, updated: 6, ready: 4, unavailable: 2)]
	clusteroperator/network is progressing: DaemonSet "/openshift-multus/network-metrics-daemon" is not available (awaiting 2 nodes)
DaemonSet "/openshift-ovn-kubernetes/ovnkube-master" is not available (awaiting 1 nodes)
```
/retest

**Comment by jhadvig** [comment]
same error as above
/retest

**Comment by jhadvig** [comment]
-retest

**Comment by TheRealJon** [comment]
TestEditUnmanagedConfigMap and TestEditUnmanagedPodDisruptionBudget cases frequently fail in `e2e-aws-operator` jobs. I'll look closer to see if there is a regression, although it doesn't seem like this change should affect those tests.

**Pull request merged**